cmd/xl-storage.go

```diff
@@ -1020,7 +1021,7 @@ func (s *xlStorage) DeleteVersion(ctx context.Context, volume, path string, fi F
 		}
 	}
 
-	if !lastVersion {
+	if !lastVersion || fi.TransitionStatus != "" {
```

Comment:
Add some more context why we are interested in TransitionStatus?

Reply:
done